docs/standard/commandline/syntax.md

```diff
@@ -0,0 +1,577 @@
+---
+title: Command-line syntax overview for System.CommandLine
+description: "An introduction to the command-line syntax that the System.CommandLine library recognizes by default. Mentions exceptions where syntax in the .NET CLI differs. Provides guidance for designing a command-line interface."
+ms.date: 02/22/2022
+no-loc: [System.CommandLine]
+helpviewer_keywords:
+  - "command line interface"
+  - "command line"
+  - "System.CommandLine"
+ms.topic: conceptual
+---
+
+# Command-line syntax overview for System.CommandLine
+
+[!INCLUDE [scl-preview](../../../includes/scl-preview.md)]
+
+This article explains the command-line syntax that `System.CommandLine` recognizes. The information will be useful to users as well as developers of .NET command-line apps, including the [.NET CLI](../../core/tools/index.md).
+
+## Tokens
+
+`System.CommandLine` parses command-line input into *tokens*, which are strings delimited by spaces. For example, consider the following command line:
+
+```dotnetcli
+dotnet tool install dotnet-suggest --global --verbosity quiet
+```
+
+This input is parsed into tokens `tool`, `install`, `dotnet-suggest`, `--global`, `--verbosity`, and `quiet`.
+
+Tokens are interpreted as commands, options, or arguments. The command-line app that is being invoked determines how the tokens after the first one are interpreted. The first token identifies the app. The following table shows how `System.CommandLine` interprets the preceding example:
+
+| Token            | Parsed as                         |
+|------------------|-----------------------------------|
+| `dotnet`         | Root command - runs *dotnet.exe*  |
```

Comment:
dotnet appearing here seems a bit confusing to someone with a tenuous grasp on what a token is.